web/libs/datamanager/src/stores/Tabs/store.js

```diff
@@ -456,6 +458,9 @@ export const TabStore = types
             field: columnID,
             schema: col.schema ?? null,
           });
+          // if (col.internal) {
+          //   return;
+          // }
```

Comment:
```suggestion
```

Reply:
still a WIP, keeping this around until the final pass in case I need it later :)